cinder/README.md

```diff
@@ -0,0 +1,209 @@
+
+#### Deploy the configured changes
+
+```bash
+(undercloud) $ openstack overcloud deploy --templates \
+  -e /home/stack/templates/overcloud_images.yaml \
+  -e /home/stack/templates/custom-dellemc-container.yaml \
+  -e /home/stack/templates/cinder-backend-dellemc-vnx.yaml \
+  -e <other templates>
+```
+
+The sequence of `-e` matters, Make sure the `/home/stack/templates/custom-vnx-container.yaml` appears after the `/home/stack/templates/overcloud_images.yaml`, so that custom VNX container can be used instead of the default one.
+
+#### Verify the configured changes
+
+After the deployment finishes successfully, in the Cinder container, the `/etc/cinder/cinder.conf` should reflect the changes made above.
+
+```ini
+[DEFAULT]
+...
+enabled_backends=tripleo_dellemc_vnx
+...
+[tripleo_dellemc_vnx]
+default_timeout=3600
+max_luns_per_storage_group=255
+naviseccli_path=/opt/Navisphere/bin/naviseccli
+san_ip=192.168.1.50
+san_login=admin
+san_password=password
+storage_vnx_pool_names=
+volume_backend_name=tripleo_dellemc_vnx
+volume_driver=cinder.volume.drivers.emc.vnx.driver.EMCVNXDriver
+storage_protocol=iscsi
+initiator_auto_registration=True
+storage_vnx_authentication_type=global
+storage_vnx_security_file_dir=
+```
+
+## Unity deployment
+
+### Prerequisites
+
+- RedHat OpenStack Platform 13.
+- Unity with version 4.1 or above.
+
+### Steps
+
+Unity is almost same as VNX, the only difference is you need to prepare Unity specific yaml.
+
+#### Prepare environment yaml for backend
+
+Copy Unity configuration template `cinder-dellemc-unity-config.yaml` from `tripleo-heat-templates`.
+
+```bash
+cp /usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-unity-config.yaml /home/stack/templates/
+```
+
+Modify backend configuration in `cinder-dellemc-unity-config.yaml`.
+
+```yaml
+# A Heat environment file which can be used to enable a
+# Cinder Dell EMC Unity backend, configured via puppet
+resource_registry:
+  OS::TripleO::Services::CinderBackendDellEMCUnity: /usr/share/openstack-tripleo-heat-templates/puppet/services/cinder-backend-dellemc-unity.yaml
+
+parameter_defaults:
+  CinderEnableIscsiBackend: false
+  CinderEnableRbdBackend: false
+  CinderEnableNfsBackend: false
+  NovaEnableRbdBackend: false
+  GlanceBackend: file
+  CinderEnableDellEMCUnityBackend: true
+  CinderDellEMCUnityBackendName: 'tripleo_dellemc_unity'
+  CinderDellEMCUnitySanIp: '192.168.1.50'
+  CinderDellEMCUnitySanLogin: 'admin'
+  CinderDellEMCUnitySanPassword: 'password'
+  CinderDellEMCUnityStorageProtocol: 'iSCSI'
+  CinderDellEMCUnityIoPorts: ''
+  CinderDellEMCUnityStoragePoolNames: ''
+```
+
+For a full detailed instruction of options, please refer to [Unity backend configuration](https://docs.openstack.org/cinder/queens/configuration/block-storage/drivers/dell-emc-unity-driver.html)
```

Comment:
The same as the previous one.
https://docs.openstack.org/cinder/latest/configuration/block-storage/drivers/dell-emc-unity-driver.html

Reply:
Done